model/v1/ids.go

```diff
@@ -124,7 +124,10 @@ func marshalBytes(dst []byte, src []byte) (n int, err error) {
 // Example: {high:2, low:1} => "AAAAAAAAAAIAAAAAAAAAAQ==".
 func (t TraceID) MarshalJSON() ([]byte, error) {
 	var b [16]byte
-	t.MarshalTo(b[:]) // can only error on incorrect buffer size
+	_, err := t.MarshalTo(b[:]) // can only error on incorrect buffer size
+	if err != nil {
+		return nil, err
+	}
```

Comment:
this is unnecessary change, the error can never happen or be tested for code coverage